third_party/NordicSemiconductor/nrfx/hal/nrf_ecb.c

```diff
@@ -66,6 +67,7 @@ bool nrf_ecb_crypt(uint8_t * dest_buf, const uint8_t * src_buf)
       return false;
```

Comment:
Do we need to call `NRFX_CRITICAL_SECTION_EXIT()` before returning false.

Reply:
Yes, good catch.